.circleci/config.yml

```diff
@@ -77,5 +77,3 @@ workflows:
                   ignore: /.*/
               requires:
                 - build
-
```

Comment:
My editor enforces the single newline at the end of the files on save. I hope this isn't an issue.